cltk/tests/test_languages/test_old_norse.py

```diff
@@ -173,6 +173,17 @@ def test_syllable_length_3(self):
             lengths.append(ont.measure_old_norse_syllable(syllable))
         self.assertListEqual(lengths, [None])
 
+    def test_prosody_tools_1(self):
+        pt = PoetryTools()
+        helgar = PoeticWord("helgar")
+        helgar.apply_poetic_tool(pt)
```

Comment:
Your API here is cool. But to justify two classes (`PoetryTools` and `PoeticWord`) here, we need to hear more from you about your long-term intentions for each. For example, will `PoetryTools` be used to specify different processing pipelines for Old Norse? or for different languages?

I hope I am making sense here. I am critiquing your code from the outside-in, but for me (at least) this is the easiest way to learn.

Reply:
In order to show why this is useful, I can make a Jupyter notebook. However this code is probably too specific and is not very useful for cltk code code. 

The "poetry tools", or whatever we call this, could be used to extract features from verses and annotate them. In Old Norse (and other old Germanic languages), there is no certain way to scan verses and classify them.